Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -830,14 +840,20 @@ private extension SPNoteListViewController {
             self?.togglePinnedState(note: note)
         }
 
+        let select = UIAction(title: ActionTitle.select, image: .image(name: .success)) { [weak self] _ in
+            self?.setEditing(true, animated: true)
+            self?.tableView.selectRow(at: indexPath, animated: false, scrollPosition: .bottom)
```

Comment:
I checked with @SylvesterWilmott and we don't want to always scroll selected row to the bottom.

I guess that you wanted to fix this issue: https://github.com/Automattic/simplenote-ios/pull/1328#issuecomment-862232153 and I would expect `scrollPosition: .none` to work, but for some reason it doesn't :-(

I tried a few things and the following combo worked, it only scrolls the row if it is covered by the bottom toolbar, otherwise scroll position is kept intact

```
self?.tableView.selectRow(at: indexPath, animated: false, scrollPosition: .none)
self?.tableView.scrollToNearestSelectedRow(at: .none, animated: false)
```

Reply:
Great.  TY!